endpoints/openrtb2/sample-requests/invalid-stored/bad_incoming_1.json

```diff
@@ -11,5 +11,5 @@
     }
   },
   "expectedReturnCode": 400,
-  "expectedErrorMessage": "Invalid request: expect { or n, but found"
+  "expectedErrorMessage": "Invalid request: Invalid JSON in Incoming Request: invalid character ']' looking for beginning of value at offset 18"
```

Comment:
As an aside, we had discussed returning generic json parse error messages to avoid test churn and to avoid having the response message deeply tied to the json parsing library. Would be good for the PMC to discuss since this is already a breaking change. Not necessarily a requirement of this PR.